tests/system/Database/Live/ModelTest.php

```diff
@@ -1461,6 +1461,44 @@ public function testGetValidationMessagesForReplace()
 
 	//--------------------------------------------------------------------
 
+	public function testSaveNewEntityWithReference()
+	{
+		$entity    = new class extends Entity{
```

Comment:
This Entity class is not updated to the [current Entity structure](https://github.com/codeigniter4/CodeIgniter4/pull/2002) and needs to be updated.

Reply:
@Tada5hi basically drop all the explicit field names and `$_options` and add:
```
protected $dates = ['created_at', 'updated_at', 'deleted_at'];
```